### ISSUE TITLE
UI/UX feedback

### ISSUE THREAD

**Issue by jdkent** [GiraffeTools/GiraffeTools#127]
Hi!

After CoastalCoding, I presented this tool in our neuroimaging workgroup meeting and got some feedback on their experience with the tool (I shared a link to the tool a few days before the meeting, but did not talk much about it until the actual meeting).

Below is the aggregated feedback from the group (I understand some of this stuff is currently being worked on/mention in other contexts so some of the feedback may not be applicable):
- A question arose whether the landing page for GiraffeTools should also prominently display Porcupine and armadillo to keep the confusion to a minimum what the person should be interacting with (a giraffe, a porcupine, or an armadillo).
- All the repositories do not show up on the projects page (I have over 80 repositories associated with my profile, and currently to add a giraffe project I need to give it a name close to the top of the alphabet, see my [giraffe projects page](https://giraffe.tools/github/jdkent) and my [github repositories](https://github.com/jdkent?tab=repositories).
- It was not obvious how to commit the workflow once the workflow was changed (had to scroll down to find the `save to github` option.
- Connecting the nodes was a little finicky (had to place the mouse in just the right place)
- I forgot how to delete a connection (this was probably just a me problem since I didn't see the trashcan until someone told me to look there, my first intuition was to click on the object and try to hit the delete key)

We think this is an excellent educational tool and will help others adopt nipype that are a bit hesitant to learn how to work with all the interfaces and workflows. We hope the feedback is helpful!

**Comment by TimVanMourik** [comment]
Hit that delete key! Implemented in 0f79074.

**Comment by TimVanMourik** [comment]
Infinite scrolling implemented! (commit ffc94a1)

**Comment by TimVanMourik** [comment]
Hi @jdkent,

I'm rounding up old issues and just wanted to give you a quick feedback on what I did with your comments:

* The whole porcupine thing is now aliased to 'workflow'. So the interface is now accessible by
https://giraffe.tools/workflow/TimVanMourik/SomeGiraffeExample
* Infinite scrolling was added a while ago
* The sidebar on the left has now a scrollable middle, and the 'Save to GitHub' is always visible
* The selection area around a port was substantially increased
* The delete/backspace key delete any selected object.

And for the rest everything should be *a lot* more stable and several features are added, like post-it notes, saving the workflow to vector image (svg) and more.

Thank you very much for the feedback!
